docs/sdk/server-side-sdks/java-cloud/java-cloud.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Java SDK for Cloud Bucketing
+---
+
+Welcome to the DevCycle Java SDK, which interfaces with the [DevCycle Bucketing API](/bucketing-api/#tag/devcycle).
+All requests, including user data are sent to DevCycle servers to ensure the User is bucketed correctly and will receive the correct variation.
+
+The SDK is available as a package on MavenCentral. It is also open source and can be viewed on Github.
+
+[![Maven](https://badgen.net/maven/v/maven-central/com.devcycle/java-server-sdk)](https://search.maven.org/artifact/com.devcycle/java-server-sdk)
+[![GitHub](https://img.shields.io/github/stars/devcyclehq/java-server-sdk.svg?style=social&label=Star&maxAge=2592000)](https://github.com/DevCycleHQ/java-server-sdk)
+
+## Requirements
```

Comment:
I think we want requirements in the 'installation' section - I believe it is also info we want pop up in context when installing (at least, that's what I've done on some other sections I've worked on).

Reply:
oh true that makes sense, i'll move it